whitelist/recipes-whitelist.txt

```diff
@@ -1,49 +1,51 @@
-# packages in core-image-minimal
-base-files
-base-passwd
-binutils-cross-arm
-core-image-minimal
-cross-localedef
-cwautomacros
-depmodwrapper-cross
-dwarfsrcfiles
-eudev
-gcc-cross-arm
-gcc-runtime
-gettext-minimal
```

Comment:
why are these files removed?

Reply:
This section now only contains conf or script only package. These removed packages contains some binaries, or already debianized thus not necessarily listed.

Reply:
I understand your intention. But could you remove only debianized packages? I'd like to keep other packages listed here.